docs/src/snap/reference/certificates.md

```diff
@@ -0,0 +1,45 @@
+# Cluster Certificates
+
+## **CertificateAuthorities (CA)**
+
+| **Common Name**              | **Usage**                            | **Path on Disk**                         | **Used For**                                |
+|------------------------------|--------------------------------------|------------------------------------------|---------------------------------------------|
+| `kubernetes-ca`               | General Kubernetes CA               | `/etc/kubernetes/pki/ca.crt`             | Signing all Kubernetes-related certificates |
+| `kubernetes-front-proxy-ca`   | CA for front-end proxy              | `/etc/kubernetes/pki/front-proxy-ca.crt` | Signing certificates for the front-proxy    |
+| `client-ca`                   | CA for client certificates          | `/etc/kubernetes/pki/client-ca.crt`      | Signing certificates for the client         |
+
+
+## **Certificates**
+
+| **Common Name**                            | **Usage** | **Path on Disk**                                     | **Used For**                                                     | **Signed By**               |
```

Comment:
```suggestion
| **Certificate Name**                            | **Purpose** | **File Location**                                     | **Primary Function**                                                     | **Signing Authority**               |
```

Reply:
Same as above (common name has a specific meaning in the context of certificates)